src/app/ApplicationTemplate.Tests/Framework/DataLoader/OnDataLostDataLoaderStrategyShould.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Chinook.DataLoader;
+using FluentAssertions;
+using Xunit;
+
+namespace ApplicationTemplate.Tests.Framework
+{
+	public class OnDataLostDataLoaderStrategyShould
+	{
+		[Fact]
+		public async Task NotDisposePreviousData_When_FirstLoad()
```

Comment:
I figure it would've been more explicit with the name, but it can definitely be reduced to like 2 tests (first load and multiple loads)

Reply:
I actually needed to use ClassData, because I didn't know how to pass instances in the inlineData, but it's fixed. I've simplified it into two unit tests like I said ^